apps/google-analytics-4/frontend/src/components/config-screen/GoogleAnalyticsConfigPage/GoogleAnalyticsConfigPage.spec.tsx

```diff
@@ -75,20 +70,17 @@ describe('Config Screen component (not installed)', () => {
   });
 
   it('prevents the app from being installed with invalid service key file', async () => {
-    const user = userEvent.setup();
     render(<GoogleAnalyticsConfigPage />);
 
     const keyFileInputBox = screen.getByLabelText(/Service Account Key/i);
 
     // user.type() got confused by the JSON string chars, so we'll just click and paste -- this
     // actually better recreates likely user behavior as a bonus
+    const user = userEvent.setup({ delay: null });
```

Comment:
Why `delay: null` here? I see https://testing-library.com/docs/user-event/options but it's not clear to me why we're disabling timeouts here (or if it's truly needed). Maybe remove or add comment.